### DISCUSSION TITLE
Понимание инъекции зависимостей ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#47] [Announcements]
# Понимание инъекции зависимостей ⚡️ Angular с примерами кода

Справочник и документация с примерами кода - Angular

https://angdev.ru/angular/dependency-injection/

**Comment by Mikman83** [comment]
Инжектирование зависимости в класс через конструктор это не единственный способ, о чем автор, вероятно, говорит выражением «-Самый распространенный способ внедрения зависимости». Но есть и не особо менее распространеный метод через декоратор @Inject. Было бы круто все варианты описать)

**Reply by bndby** [reply]
Согласен. Но я давно с ангуляром не работаю, поэтому буду признателен, если дадите ссылку на доку, чтобы я мог дополнить этот момент
